test/e2e/framework/util.go

```diff
@@ -928,7 +928,7 @@ func deleteNS(c clientset.Interface, clientPool dynamic.ClientPool, namespace st
 	}
 
 	// wait for namespace to delete or timeout.
-	err := wait.PollImmediate(5*time.Second, timeout, func() (bool, error) {
+	err := wait.PollImmediate(Poll, timeout, func() (bool, error) {
```

Comment:
This was just cleanup, it's non-essential; I noticed namespace deletion was a bit slow so was trying to tighten things up a bit. If this is risky I can undo.